src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: "Avenir-book";
+  src: url(./fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: "Avenir-heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+.big-form,
+.small-form {
+  margin-top: 20px;
+}
+
+.big-input {
+  width: 100%;
+  height: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  padding-left: 62px;
+}
+
+.common-input {
+  font-family: Avenir-book, sans-serif;
```

Comment:
add font-family to html and use `inherit` here
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit

Reply:
Understandable

Reply:
@font-face {
  font-family: "Avenir-book";
  src: url(./fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: "Avenir-Heavy";
  src: url(./fonts/Avenir-Heavy.ttf);
}

html {
  font-family: Avenir, sans-serif;
  font-weight: 300px;
}

and after 
 .common-input {
  font-family: inherit;
}

Like this?

Reply:
yep, like this